jipdate.py

```diff
@@ -227,7 +232,7 @@ def parse_status_file(jira, filename):
     # [SWG-28]
     # [LITE-32]
     # etc ...
-    regex = r"^\[([A-Z]+-\d+)\]\n$"
+    regex = r"^\[(.*)\]\n$"
```

Comment:
Yes. What we generally do in our weekly status is to start a new section by [ABCD] with ABCD being an issue or anything:

[PMWG-12345]
Sent a patch on mailing list for review

[Misc]
I will be on holidays next week

[jipdate]
Proposed some changes so we can be more permissive in the content of the file